test/statmodels.jl

```diff
@@ -18,6 +18,14 @@ x1    1.45666         Good    1  -12.56  0.1200
 x2  -23.14            Great  56    0.13  0.3467
 x3    1.56734e-13     Bad     2    0.00  <1e-15
 ───────────────────────────────────────────────"""
+
+@test sprint(show, MIME"text/markdown"(), ct) == """
+|    |     Estimate  |Comments  |df  |     t  |     p  |
```

Comment:
Sorry I hadn't noticed this: why add two spaces after the column names rather than a single one? That would align them with the contents of the columns.

Reply:
I think I've fixed this, but if it becomes too painful, I'll swap over to using the `Markdown` stdlib (which would lose the decimal alignment but reduce complexity).

Reply:
This is OK I think.